src/PropertyInfo/TypeInfoParsers/NativeTypeInfoParser.php

```diff
@@ -19,11 +17,8 @@
  *
  * @author Mihai Stancu <stancu.t.mihai@gmail.com>
  */
-abstract class NativeTypeInfoParser implements TypeInfoParserInterface
+trait NativeTypeInfoParser
```

Comment:
Yes, I already did -- with the commits I pushed a few minutes ago.

Unfortunately this branch was meant to extend the TypeInfoParser to apply it to the DoctrineExtractor so there may be some conflicts between this PR and that one.